doc/building.md

```diff
@@ -61,16 +47,34 @@ will build out of the box without further effort:
 
 ### Compile
 
-    mkdir cmake-build
-    cd cmake-build
+Add bincrafters repository to conan:
+
+    conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+
+In general to build you need to:
+
+    conan install -if build --build missing .
+    conan build -bf build -sf . .
+
```

Comment:
so what is the difference if I build using cmake ../ vs conan build ?

Reply:
Here is about 3 ways to build project with conan:
1. `conan install <...> && conan build <..>`. In such way will be called conan installed in the system. Project will be built in release mode with threads(like `make -j`).
2. `cmake .. && cmake --build .`. That way will download conan linke cmake script, you can find link in CMakeLists.txt . I found only one problem with such conan: it doesn't support Apple Clang 12.0
3. `conan install <...> && cmake --build .`. It will call conan like in 2, but libraries are already installed.

If it first build of project prefer `1`, but after it you can use all that you want.